File: remappedSrc/net/fabricmc/example/ExampleModClient.java
package net.fabricmc.example;

import net.fabricmc.api.ClientModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class ExampleModClient implements ClientModInitializer {
    public static final Logger LOGGER = LoggerFactory.getLogger("modid");

    @Override
    public void onInitializeClient() {
        LOGGER.info("Hello Fabric world!");
    }
}

File: src/main/java/net/fabricmc/example/ExampleModClient.kt
package net.fabricmc.example

import net.fabricmc.api.ClientModInitializer
import net.fabricmc.example.ExampleModMain.Companion.MOD_ID
import org.slf4j.Logger
import org.slf4j.LoggerFactory

class ExampleModClient: ClientModInitializer {
    companion object{
        val LOGGER: Logger = LoggerFactory.getLogger(MOD_ID)
    }

    override fun onInitializeClient() {
        LOGGER.info("Hello Fabric world!")
    }
}